modules/ocf_desktop/files/xsession/staff-logout

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import datetime
+from subprocess import call
+from subprocess import PIPE
+from subprocess import Popen
+from time import sleep
+
+from ocflib.lab.hours import Day
+
+# delay before killing session in minutes
+delay = 45
+# extra delay if we will be closed delay minutes from now
+bonus = 30
+now = datetime.datetime.now()
+if not Day.from_date(now).is_open(now + datetime.timedelta(minutes=delay)):
+    delay += bonus
+
+sleep(60 * delay)
+
+
+cmd = 'xscreensaver-command -display :0.0 -time'.split()
+p = Popen(cmd, stdout=PIPE)
```

Comment:
don't split the string like that :(

if you really insist on splitting, do it properly with `shlex.split` which understands shell syntax. preferably just write out the arguments:
```python
proc = subprocess.Popen(
    ('xscreensaver-command', '-display', ':0.0', '-time'),
    stdout=subprocess.PIPE,
)
```

however you don't need Popen here at all

```python
output = subprocess.check_output(('xscreensaver-command', '-display', ':0.0', '-time'))
```